dags/glam_fenix.py

```diff
@@ -159,9 +159,21 @@
         scalar_bucket_counts = query(task_name=f"{product}__scalar_bucket_counts_v1")
         scalar_probe_counts = query(task_name=f"{product}__scalar_probe_counts_v1")
 
-        histogram_bucket_counts = query(
-            task_name=f"{product}__histogram_bucket_counts_v1"
-        )
+        with TaskGroup(
+            group_id=f"{product}__histogram_bucket_counts_v1", dag=dag, default_args=default_args
+        ) as histogram_bucket_counts:
+            prev_task = None
+            for sample_range in ([0, 19], [20, 39], [40, 59], [60, 79], [80, 99]):
```

Comment:
nit: I'd probably change this to something like:
```
for sample_low, sample_high in ([0, 19], [20, 39], [40, 59], [60, 79], [80, 99]):
    ...
```

So the indexing below is a little clearer

Reply:
That's a good idea, though I'd suggest the variable names `min_sample_id` and `max_sample_id` to match the existing nomenclature.